docs/reference/release-notes/7.2.asciidoc

```diff
@@ -0,0 +1,481 @@
+[[release-notes-7.2.0]]
+== {es} version 7.2.0
+
+Also see <<breaking-changes-7.2,Breaking changes in 7.2>>.
+
+[[breaking-7.2.0]]
+[float]
+=== Breaking changes
+
+Cluster Coordination::
+* Reject port ranges in `discovery.seed_hosts` {pull}41404[#41404] (issue: {issue}40786[#40786])
+
+
+[[breaking-java-7.2.0]]
+[float]
+=== Breaking Java changes
+
+Infra/Plugins::
+* Remove IndexStore and DirectoryService {pull}42446[#42446]
+
+
+[[deprecation-7.2.0]]
+[float]
+=== Deprecations
+
+Authorization::
+* Deprecate permission over aliases {pull}38059[#38059]
+
+Features/Features::
+* Add deprecation check for ILM poll interval <1s {pull}41096[#41096] (issue: {issue}39163[#39163])
+
+Mapping::
+* Enforce Completion Context Limit {pull}38675[#38675] (issue: {issue}32741[#32741])
+
+Reindex::
+* Reindex from remote deprecation of escaped index {pull}41005[#41005] (issue: {issue}40303[#40303])
+
+Search::
+* Deprecate using 0 value for `min_children` in `has_child` query #41548 {pull}41555[#41555] (issue: {issue}41548[#41548])
+* Deprecate support for first line empty in msearch API {pull}41442[#41442] (issue: {issue}41011[#41011])
+
+Security::
+* Deprecate the native realm migration tool {pull}42142[#42142]
+
+[[feature-7.2.0]]
+[float]
+=== New features
+
+Authentication::
+* Add an OpenID Connect authentication realm {pull}40674[#40674]
+
+Cluster Coordination::
+* Add GET /_cluster/master endpoint {pull}40047[#40047]
```

Comment:
This PR was closed, not merged. I suspect it appears here because I didn't remove the version labels, sorry.

Reply:
thanks! removed in https://github.com/elastic/elasticsearch/pull/43343/commits/f93cf4fc23d424131e6c3d772dbfa84043402b7f